README.md

```diff
@@ -60,70 +68,70 @@ sudo sh -c 'curl -L https://github.com/redhat-developer/odo/releases/download/v0
 </details>
 
 <details>
-<summary> :package: :checkered_flag: Windows</summary>
-
-#### In order to correctly use odo you must add it to your PATH environment variable
+<summary> Windows</summary>
 
 1. Download the `odo-windows-amd64.exe.gz` file from the [GitHub releases page](https://github.com/redhat-developer/odo/releases).
-2. Extract the file
+2. Extract the file.
 3. Add the location of extracted binary to your PATH environment variable by following [this Wiki page](https://github.com/redhat-developer/odo/wiki/Setting-PATH-variable-on-Windows).
 
 </details>
 
-#### For a list of other methods such as installing the latest mastery binary, or specific OS installations, visit the [installation page](/docs/installation.md).
-
-## Purpose
+For a list of other methods such as installing the latest binary on specific OS installations, visit the [installation page](/docs/installation.md).
 
-OpenShift Do (odo) is a CLI tool for developers who are writing, building, and deploying applications on OpenShift. With odo, developers get an opinionated CLI tool that supports fast, iterative development which abstracts away Kubernetes and OpenShift concepts, thus allowing them to focus on what's most important to them: code.
-
-Odo was created to improve the developer experience with OpenShift. Existing tools such as `oc` are more operations-focused and requires a deep-understanding of Kubernetes and OpenShift concepts. Odo is designed to be simple and concise so you may focus on coding rather than how to deploy your application. Since odo can build and deploy your code to your cluster immediately after you save you changes, you benefit from instant feedback and can thus validate your changes in real-time. Odo's syntax and design is centered around concepts already familiar to developers, such as: project, application and component.
-
-## Demo
+## odo overview
 
 [![asciicast](https://asciinema.org/a/225717.svg)](https://asciinema.org/a/225717)
 
-## Deploying an Application using odo
+## Deploying an application using odo
 
-After you have odo installed, follow these steps to build, push, and deploy a Node.js application using odo. Examples for other supported languages and runtimes can be found [here](https://github.com/redhat-developer/odo/blob/master/docs/examples.md).
-
-```sh
+After you install odo, follow these steps to build, push, and deploy a Node.js application using odo. Examples for other supported languages and runtimes can be found [here](https://github.com/redhat-developer/odo/blob/master/docs/examples.md).
```

Comment:
Change to "After installing odo, follow these steps to build, push, and deploy a Node.js application."